src/Backend/Modules/MailMotor/MailMotor.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Backend\Modules\MailMotor;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Backend\Modules\MailMotor\DependencyInjection\Compiler\CustomCompilerPass;
+
+class MailMotor extends Bundle
+{
+    public function build(ContainerBuilder $container)
+    {
+        parent::build($container);
+
+        $container->addCompilerPass(new CustomCompilerPass());
```

Comment:
Wouldn't `MailMotorCompilerPass` be a better name instead of `CustomCompilerPass`?